mattermost-plugin/webapp/src/error_boundary.tsx

```diff
@@ -2,27 +2,43 @@
 // See LICENSE.txt for license information.
 
 import React from 'react'
+import PropTypes from 'prop-types'
+
+import {Utils} from '../../../webapp/src/utils'
 
-type Props = {}
 type State = {
     hasError: boolean
 }
 
-export default class ErrorBoundary<Props, State> extends React.Component {
+export default class ErrorBoundary extends React.Component {
     state = {hasError: false}
+    propTypes = {children: PropTypes.node.isRequired}
+    msg = 'Redirecting to error page...'
+
+    handleError = (): void => {
+        const url = Utils.getBaseURL() + '/error?id=unknown'
+        Utils.log('error boundary redirecting to ' + url)
+        window.location.replace(url)
+    }
 
-    static getDerivedStateFromError(error: Error) {
+    static getDerivedStateFromError(/*error: Error*/): State {
```

Comment:
nit: should this commented section be removed.

Reply:
This is commented out to avoid linter warning about unused param.  Leaving the param there helps readers understand that a param is provided and what type.  It's like the `_` for unused params in Go which still let you see what was passed.